src/aatoolbox/config/pathconfig.py

```diff
@@ -0,0 +1,22 @@
+"""Path configuration."""
+
+import os
+from dataclasses import dataclass
+from pathlib import Path
+
+BASE_DIR_ENV = "AA_DATA_DIR"
```

Comment:
How does this work? Is it set here like this because I am able to manually change this in a session or script?

```python
import aatoolbox.config.pathconfig

BASE_DIR_ENV = "OTHER_ENV_VAR"
```

And then I can init a `PathConfig()` with my `"OTHER_ENV_VAR"` as the default? Is there a reason not to just pass in the name of the environment variable directly into the initialisation, and the user just has to specify their environment variable? 

Or is this variable being set just for cleanliness of code in the module and if we change the name of the env var from `"AA_DATA_DIR"` then it's easier for us to just change at the top of the script? 

Sorry, think I'm not getting exactly how this works, so interested to see your thoughts!

Reply:
I set it there because I thought it makes sense (at least for now) as a module-level [constant](https://www.python.org/dev/peps/pep-0008/#constants)

Reply:
Clear. Since it's intended to be non-public, should we put leading underscores in front of it as described [here](https://www.python.org/dev/peps/pep-0008/#global-variable-names)? I just came across that actually on something I was reviewing of Tinka's!

Reply:
That's a good point. We should really setup `__all__` in one of the init's but I will add the underscores for now!